src/_P097_Esp32Touch.ino

```diff
@@ -0,0 +1,289 @@
+#ifdef USES_P097
+
+// #######################################################################################################
+// #################################### Plugin 097: ESP32 Touch ##########################################
+// #######################################################################################################
+
+
+#ifdef ESP32
+
+
+# define PLUGIN_097
+# define PLUGIN_ID_097         97
+# define PLUGIN_NAME_097       "Touch (ESP32) - internal"
+# define PLUGIN_VALUENAME1_097 "Touch"
+
+# include "_Plugin_Helper.h"
+
+# ifdef ESP32
+  #  define P097_MAX_ADC_VALUE    4095
+# endif // ifdef ESP32
+# ifdef ESP8266
+  #  define P097_MAX_ADC_VALUE    1023
+# endif // ifdef ESP8266
+
+
+# define P097_SEND_TOUCH_EVENT    PCONFIG(0)
+# define P097_SEND_RELEASE_EVENT  PCONFIG(1)
+# define P097_SEND_DURATION_EVENT PCONFIG(2)
+# define P097_TOUCH_THRESHOLD     PCONFIG(3)
```

Comment:
@TD-er maybe it could be also bitfield?

Reply:
Was thinking of it, but I was not yet sure whether it would stay a binary value or some combobox.